File: src/main/java/me/diax/diax/commands/information/Invite.java
package me.diax.diax.commands.information;

import me.diax.comportment.jdacommand.Command;
import me.diax.comportment.jdacommand.CommandAttribute;
import me.diax.comportment.jdacommand.CommandDescription;
import net.dv8tion.jda.core.Permission;
import net.dv8tion.jda.core.entities.Message;

@CommandDescription(
        name = "invite",
        triggers = "invite",
        attributes = {
                @CommandAttribute(key = "private")
        }
)
public class Invite implements Command {

    @Override
    public void execute(Message trigger, String args) {
        trigger.getChannel().sendMessage(trigger.getJDA().asBot().getInviteUrl(Permission.MESSAGE_EMBED_LINKS, Permission.VOICE_CONNECT, Permission.VOICE_SPEAK, Permission.MESSAGE_READ, Permission.MESSAGE_WRITE, Permission.MESSAGE_ADD_REACTION, Permission.MESSAGE_ATTACH_FILES)).queue();
    }
}

File: src/main/kotlin/me/diax/diax/commands/information/Invite.kt
package me.diax.diax.commands.information

import me.diax.comportment.jdacommand.Command
import me.diax.comportment.jdacommand.CommandDescription
import net.dv8tion.jda.core.entities.Message

@CommandDescription(
    name = "invite",
    triggers = ["invite"]
)
class Invite : Command {

    override fun execute(trigger: Message, args: String) {
        trigger.channel.sendMessage("https://discordapp.com/oauth2/authorize?scope=bot&client_id=295500621862404097&permissions=3198016").queue()
    }
}